src/safari/safari.xcodeproj/xcuserdata/alecmaliwanag.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should gitignore more files:
https://github.com/github/gitignore/blob/master/Global/Xcode.gitignore